spec/gameobj-data/boxes_and_picking_spec.rb

```diff
@@ -57,6 +57,7 @@
       %{stained},
     ] # Similar to box_metals or box_woods
 
+    # Leaving in descriptions in case they are used in the future
     boon_box_long = [
```

Comment:
If you remove the examples using the long descriptions, please remove this definition as well so that future readers don't think it is load bearing.